components/input/input.vue

```diff
@@ -45,9 +44,7 @@
         <div
           v-if="shouldValidateLength"
           data-qa="dt-input-length-description"
-          :class="[
-            'd-mb2',
-          ]"
+          class="d-input__length-description"
```

Comment:
Those were not necessary, I added them previously to dialtone because I thought `d-description` and `d-label` were utilities, but they are not. Will remove them from dialtone.